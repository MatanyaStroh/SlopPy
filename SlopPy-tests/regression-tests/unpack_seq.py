(x, y, z) = "hello,world".split(',')
assert type(x) is NA
assert type(y) is NA
assert type(z) is NA
