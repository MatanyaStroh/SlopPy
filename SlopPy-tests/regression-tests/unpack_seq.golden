{NA <type 'exceptions.ValueError'> 'need more than 2 values to unpack'
 Traceback (most recent call last):
  File "regression-tests/unpack_seq.py", line 1, in <module>
    (x, y, z) = "hello,world".split(',')
}
